templates/execution/address.html

```diff
@@ -123,6 +123,16 @@
         font-size: 1.2rem;
       }
     }
+    @media screen and (max-width: 360px) {
+      .header-address .text-monospace {
+        font-size: 0.7rem;
+      }
+    }
+    @media screen and (max-width: 320px) {
+      .header-address .text-monospace {
+        font-size: 0.6rem;
+      }
+    }
```

Comment:
I have played around with this and suggest the following addition:
```suggestion
    @media screen and (max-width: 360px) {
      .header-address .text-monospace {
        font-size: 0.7rem;
      }
    }
    @media screen and (max-width: 320px) {
      .header-address .text-monospace {
        font-size: 0.65rem;
      }
    }
    @media screen and (max-width: 294px) {
      .header-address .text-monospace {
        font-size: 0.6rem;
      }
    }
```